bigtable-hbase-integration-tests/src/test/java/com/google/cloud/bigtable/hbase/TestImport.java

```
@@ -78,29 +84,31 @@ public void testMapReduce() throws IOException, ClassNotFoundException, Interrup
 
     // Create new table.
     TableName newTableName = IntegrationTests.newTestTableName();
-    Table newTable = getConnection().getTable(newTableName);
-
-    // Change for method in IntegrationTests
-    Admin admin = getConnection().getAdmin();
-    HColumnDescriptor hcd = new HColumnDescriptor(IntegrationTests.COLUMN_FAMILY);
-    HTableDescriptor htd = new HTableDescriptor(newTableName);
-    htd.addFamily(hcd);
-    admin.createTable(htd);
+    try (Table newTable = getConnection().getTable(newTableName)){
+      // Change for method in IntegrationTests
+      HColumnDescriptor hcd = new HColumnDescriptor(IntegrationTests.COLUMN_FAMILY);
+      HTableDescriptor htd = new HTableDescriptor(newTableName);
+      htd.addFamily(hcd);
+      admin.createTable(htd);
 
-    // Run the import.
-    args = new String[]{
-        newTableName.getNameAsString(),
-        tempDir
-    };
-    job = Import.createSubmittableJob(conf, args);
-    job.getConfiguration().set("fs.defaultFS", "file:///");
-    Assert.assertTrue(job.waitForCompletion(true));
+      // Run the import.
+      args = new String[]{
+          newTableName.getNameAsString(),
+          tempDir
+      };
+      job = Import.createSubmittableJob(conf, args);
+      job.getConfiguration().set("fs.defaultFS", "file:///");
+      Assert.assertTrue(job.waitForCompletion(true));
 
-    // Assert the value is there.
-    get = new Get(rowKey);
-    result = newTable.get(get);
-    cells = result.listCells();
-    Assert.assertEquals(1, cells.size());
-    Assert.assertArrayEquals(CellUtil.cloneValue(cells.get(0)), value);
+      // Assert the value is there.
+      Get get = new Get(rowKey);
+      Result result = newTable.get(get);
+      List<Cell> cells = result.listCells();
+      Assert.assertEquals(1, cells.size());
+      Assert.assertArrayEquals(CellUtil.cloneValue(cells.get(0)), value);
+    } finally {
+      admin.disableTable(newTableName);
```

ditto, log and ignore the error.